packages/cli/src/commands/config/set.ts

```diff
@@ -64,4 +66,25 @@ export default class Set extends Command {
       );
     }
   }
+
+  private async displayConfig(cfg: Config) {
```

Comment:
There should be a shared function between `config/get.ts` and `config/set.ts` so that we always display the same thing for both.

Reply:
Should we display the access token with the `config:get`, `config:set`, both or neither?

Reply:
If request explicitly with `get` yes (i.e. `coveo config:get access_token` (I don't recall exactly the syntax but well, you get the gist of it :P )

In other cases, we shall not.